proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,91 @@
+# cert-manager Incubation Proposal
+
+cert-manager joined the CNCF as a sandbox project just over a year ago, and we firmly believe that the project is ready to take the next step and graduate to the Incubation stage.
+
+cert-manager provides reliable, automated Kubernetes-native management of X.509 certificates. It reduces the risk of certificate-related outages and enables PKI at scale, and is treated by many teams as de-facto part of the core components of a Kubernetes or OpenShift deployment.
+
+The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.
+
+The main cert-manager repo has 8.4k stars on GitHub and hundreds of projects downstream which depend on cert-manager code in some way. There are thousands of companies which have contributed in some way and cert-manager's use and prominence remains strong.
```

Comment:
> thousands of companies which have contributed in some way

Do we want to call out companies, or just people? "thousands of people which have contributed"?

Reply:
I moved this bit from "production use" where I was talking more about companies. Definitely would read better as "people", will change :+1: